### ISSUE TITLE
Bulk Job error

### ISSUE THREAD

**Issue by mxk1235** [felixlindstrom/python-salesforce-api#11]
Seems like something is wrong with bulk jobs.  InProgress jobs get returned as errors.

```
Starting update
'InProgress' is not a valid JOB_STATE
ValueError: 'InProgress' is not a valid JOB_STATE

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "sfdc.py", line 131, in <module>
    raise e
  File "sfdc.py", line 126, in <module>
    resp = sf.bulk.update('Contact', upd)
  File "/Users/mxk/opt/anaconda3/lib/python3.8/site-packages/salesforce_api/services/bulk/v2.py", line 44, in update
    return self._execute_operation(OPERATION.UPDATE, object_name, entries)
  File "/Users/mxk/opt/anaconda3/lib/python3.8/site-packages/salesforce_api/services/bulk/v2.py", line 58, in _execute_operation
    return job.wait()
  File "/Users/mxk/opt/anaconda3/lib/python3.8/site-packages/salesforce_api/services/bulk/v2.py", line 115, in wait
    while not self.is_done():
  File "/Users/mxk/opt/anaconda3/lib/python3.8/site-packages/salesforce_api/services/bulk/v2.py", line 98, in is_done
    return self.get_state() in JOB_STATES_DONE
  File "/Users/mxk/opt/anaconda3/lib/python3.8/site-packages/salesforce_api/services/bulk/v2.py", line 95, in get_state
    return JOB_STATE(self.info().get('state'))
  File "/Users/mxk/opt/anaconda3/lib/python3.8/enum.py", line 309, in __call__
    return cls.__new__(cls, value)
  File "/Users/mxk/opt/anaconda3/lib/python3.8/enum.py", line 600, in __new__
    raise exc
  File "/Users/mxk/opt/anaconda3/lib/python3.8/enum.py", line 584, in __new__
    result = cls._missing_(value)
  File "/Users/mxk/opt/anaconda3/lib/python3.8/enum.py", line 613, in _missing_
    raise ValueError("%r is not a valid %s" % (value, cls.__name__))
ValueError: 'InProgress' is not a valid JOB_STATE
```

**Comment by kenpomd** [comment]
I encountered the same error with a bulk insert. I checked the Salesforce API bulk API 2.0 at https://developer.salesforce.com/docs/atlas.en-us.api_bulk_v2.meta/api_bulk_v2/get_job_info.htm and it looks like the possible JobState values are as follows:

- Open
- UploadComplete
- InProgress
- Aborted
- JobComplete
- Failed

The JOB_STATE class defined in v2.py does not include 'InProgress' and the ValueError exception is thrown when trying to instantiate an instance of JOB_STATE with 'InProgress' in get_state(). It looks like 'InProgress' could be added to the JOB_STATE class and then the is_done() method will not throw a ValueError exception.
